server/routes/red-flags.js

```diff
@@ -11,5 +11,9 @@ router.get('/red-flags/:id', redFlags.getSingleRedFlag);
 
 router.post('/red-flags', midware.isValid, redFlags.createRedFlag);
 
+router.patch('/red-flags/:id/location', midware.isValid, redFlags.editLocation);
```

Comment:
based on what you have in your validation middleware, are you saying `comment` must be present before the user can update the location?